MiscVBAFunctions/Modules/MiscHasKey.bas

```diff
@@ -8,43 +8,110 @@ Private Sub TestHasKey()
     c.Add col("x", "y", "z"), "b"
     
     Debug.Print vbLf & "*********** TestHasKey tests ***********"
-    Debug.Print hasKey(c, "a") ' True for scalar
-    Debug.Print hasKey(c, "b") ' True for object
-    Debug.Print hasKey(c, "A") ' False (case insensitive)
+    Debug.Print True, hasKey(c, "a") ' True for scalar
+    Debug.Print True, hasKey(c, "b") ' True for object
+    Debug.Print True, hasKey(c, "A") ' True (even though case insensitive???)
```

Comment:
Yes, I had it wrong, it will return `True` for collections as it's case insensitive, right?

Reply:
Yes it returns True on my checks